src/v2/Apps/Artist/Routes/WorksForSale/Components/ArtistArtworkFilter.tsx

```diff
@@ -98,7 +115,12 @@ const ArtistArtworkFilter: React.FC<ArtistArtworkFilterProps> = props => {
           aggregations: ["TOTAL"],
         }}
         enableCreateAlert
-        savedSearchEntity={savedSearchEntity}
+        renderFilterPills={() => (
+          <ActiveFilterPills defaultPills={defaultPills} />
+        )}
+        renderCreateAlertButton={() => (
+          <DefaultCreateAlertButton savedSearchEntity={savedSearchEntity} />
+        )}
```

Comment:
Please note that we now need to duplicate some artist info a bit and pass it for pills and create alert flow separately